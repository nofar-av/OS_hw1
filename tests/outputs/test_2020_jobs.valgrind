==26831== Memcheck, a memory error detector
==26831== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==26831== Using Valgrind-3.13.0 and LibVEX; rerun with -h for copyright info
==26831== Command: /home/omer/OS_wet1/smash
==26831== Parent PID: 26829
==26831== 
==26831== 
==26831== FILE DESCRIPTORS: 4 open at exit.
==26831== Open file descriptor 3: /home/omer/OS_wet1/tests/outputs/test_2020_jobs.valgrind
==26831==    <inherited from parent>
==26831== 
==26831== Open file descriptor 2: /home/omer/OS_wet1/tests/outputs/test_2020_jobs.err
==26831==    <inherited from parent>
==26831== 
==26831== Open file descriptor 1: /home/omer/OS_wet1/tests/outputs/test_2020_jobs.out
==26831==    <inherited from parent>
==26831== 
==26831== Open file descriptor 0:
==26831==    <inherited from parent>
==26831== 
==26831== 
==26831== HEAP SUMMARY:
==26831==     in use at exit: 0 bytes in 0 blocks
==26831==   total heap usage: 435 allocs, 435 frees, 107,022 bytes allocated
==26831== 
==26831== All heap blocks were freed -- no leaks are possible
==26831== 
==26831== For counts of detected and suppressed errors, rerun with: -v
==26831== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
