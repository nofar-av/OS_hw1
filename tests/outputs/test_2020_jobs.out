smash> smash> smash> smash> smash> smash> smash> smash> [1] sleep 100& : 28180 0 secs
[2] sleep 200& : 28181 0 secs
[3] sleep 300& : 28183 0 secs
[4] sleep 400& : 28184 0 secs
[5] sleep 20& : 28185 0 secs
[6] sleep 600& : 28186 0 secs
[7] sleep 700& : 28187 0 secs
smash> smash> smash> [1] sleep 100& : 28180 0 secs
[2] sleep 200& : 28181 0 secs
[3] sleep 300& : 28183 0 secs
[4] sleep 400& : 28184 0 secs
[5] sleep 20& : 28185 0 secs
[6] sleep 600& : 28186 0 secs
[7] sleep 700& : 28187 0 secs
[8] sleep 800& : 28188 0 secs
[9] sleep 900& : 28189 0 secs
smash> signal number 9 was sent to pid 28183
smash> [1] sleep 100& : 28180 1 secs
[2] sleep 200& : 28181 1 secs
[4] sleep 400& : 28184 1 secs
[5] sleep 20& : 28185 1 secs
[6] sleep 600& : 28186 1 secs
[7] sleep 700& : 28187 1 secs
[8] sleep 800& : 28188 1 secs
[9] sleep 900& : 28189 1 secs
smash> smash> [1] sleep 100& : 28180 1 secs
[2] sleep 200& : 28181 1 secs
[4] sleep 400& : 28184 1 secs
[5] sleep 20& : 28185 1 secs
[6] sleep 600& : 28186 1 secs
[7] sleep 700& : 28187 1 secs
[8] sleep 800& : 28188 1 secs
[9] sleep 900& : 28189 1 secs
[10] sleep 1000& : 28384 0 secs
smash> signal number 9 was sent to pid 28384
smash> [1] sleep 100& : 28180 3 secs
[2] sleep 200& : 28181 3 secs
[4] sleep 400& : 28184 3 secs
[5] sleep 20& : 28185 3 secs
[6] sleep 600& : 28186 3 secs
[7] sleep 700& : 28187 3 secs
[8] sleep 800& : 28188 3 secs
[9] sleep 900& : 28189 3 secs
smash> smash> [1] sleep 100& : 28180 3 secs
[2] sleep 200& : 28181 3 secs
[4] sleep 400& : 28184 3 secs
[5] sleep 20& : 28185 3 secs
[6] sleep 600& : 28186 3 secs
[7] sleep 700& : 28187 3 secs
[8] sleep 800& : 28188 3 secs
[9] sleep 900& : 28189 3 secs
[10] sleep 1001& : 28494 0 secs
smash> sleep 1001& : 28494
smash: got ctrl-Z
smash: process 28494 was stopped
smash> [1] sleep 100& : 28180 3 secs
[2] sleep 200& : 28181 3 secs
[4] sleep 400& : 28184 3 secs
[5] sleep 20& : 28185 3 secs
[6] sleep 600& : 28186 3 secs
[7] sleep 700& : 28187 3 secs
[8] sleep 800& : 28188 3 secs
[9] sleep 900& : 28189 3 secs
[10] sleep 1001& : 28494 0 secs (stopped)
smash> sleep 1001& : 28494
smash: got ctrl-Z
smash: process 28494 was stopped
smash> [1] sleep 100& : 28180 3 secs
[2] sleep 200& : 28181 3 secs
[4] sleep 400& : 28184 3 secs
[5] sleep 20& : 28185 3 secs
[6] sleep 600& : 28186 3 secs
[7] sleep 700& : 28187 3 secs
[8] sleep 800& : 28188 3 secs
[9] sleep 900& : 28189 3 secs
[10] sleep 1001& : 28494 0 secs (stopped)
smash> sleep 1001& : 28494
smash: got ctrl-Z
smash: process 28494 was stopped
smash> [1] sleep 100& : 28180 3 secs
[2] sleep 200& : 28181 3 secs
[4] sleep 400& : 28184 3 secs
[5] sleep 20& : 28185 3 secs
[6] sleep 600& : 28186 3 secs
[7] sleep 700& : 28187 3 secs
[8] sleep 800& : 28188 3 secs
[9] sleep 900& : 28189 3 secs
[10] sleep 1001& : 28494 0 secs (stopped)
smash> sleep 1001& : 28494
smash: got ctrl-Z
smash: process 28494 was stopped
smash> [1] sleep 100& : 28180 3 secs
[2] sleep 200& : 28181 3 secs
[4] sleep 400& : 28184 3 secs
[5] sleep 20& : 28185 3 secs
[6] sleep 600& : 28186 3 secs
[7] sleep 700& : 28187 3 secs
[8] sleep 800& : 28188 3 secs
[9] sleep 900& : 28189 3 secs
[10] sleep 1001& : 28494 0 secs (stopped)
smash> smash: got ctrl-C
[1] sleep 100& : 28180 4 secs
[2] sleep 200& : 28181 4 secs
[4] sleep 400& : 28184 4 secs
[5] sleep 20& : 28185 4 secs
[6] sleep 600& : 28186 4 secs
[7] sleep 700& : 28187 4 secs
[8] sleep 800& : 28188 4 secs
[9] sleep 900& : 28189 4 secs
[10] sleep 1001& : 28494 1 secs (stopped)
smash> smash: got ctrl-Z
smash: process 28634 was stopped
smash> [1] sleep 100& : 28180 7 secs
[2] sleep 200& : 28181 7 secs
[4] sleep 400& : 28184 7 secs
[5] sleep 20& : 28185 7 secs
[6] sleep 600& : 28186 7 secs
[7] sleep 700& : 28187 7 secs
[8] sleep 800& : 28188 7 secs
[9] sleep 900& : 28189 7 secs
[10] sleep 1001& : 28494 4 secs (stopped)
[11] sleep 1002 : 28634 0 secs (stopped)
smash> sleep 1002 : 28634
smash> [1] sleep 100& : 28180 7 secs
[2] sleep 200& : 28181 7 secs
[4] sleep 400& : 28184 7 secs
[5] sleep 20& : 28185 7 secs
[6] sleep 600& : 28186 7 secs
[7] sleep 700& : 28187 7 secs
[8] sleep 800& : 28188 7 secs
[9] sleep 900& : 28189 7 secs
[10] sleep 1001& : 28494 4 secs (stopped)
[11] sleep 1002 : 28634 0 secs
smash> smash: got ctrl-Z
smash: process 28902 was stopped
smash> 